tasks/authorized-keys.yml

```diff
@@ -5,7 +5,7 @@
     user: "{{ item.owner }}"
     key: "{{ lookup('file', item.src) }}"
     state: "{{ item.state | default('present') }}"
-    path: "~/.ssh/{{ item.path | default('authorized_keys') }}"
+    path: "{{ item_path | default('~' + item.owner + '/.ssh/authorized_keys') }}"
```

Comment:
`item.path`

Reply:
Fixed (sorry about that) wrong copy/paste.